content/new-website.rst

```diff
@@ -11,4 +11,4 @@ The libgd.org has a new website.  We have built the new site in
 view the sources used to build this website in its own `git repository`_.
 
 .. _Pelican: http://getpelican.com
-.. _git repository: https://github.org/libgd/website
+.. _git repository: https://github.com/libgd/oldwebsite
```

Comment:
it should be pointing to this repo -- `https://github.com/libgd/libgd.github.io`

Reply:
Ha, my bad, i've put this cause actual link redirect to https://github.com/libgd/oldwebsite

Reply:
that was the correct site previously (and when this news entry was written), but we've consolidated the logic